ast2vec/repo2/base.py

```diff
@@ -148,8 +163,8 @@ def _bblfsh_parse(self, thread_index, filepath, language):
         try:
             return self._bblfsh[thread_index].parse(
                 filepath, language=language, timeout=self._timeout)
-        except DecodeError as e:
-            msg = "bblfsh raised an DecodeError exception. Probably your protobuf is <= v3.3.2 " \
+        except RpcError as e:
```

Comment:
Why? It is exactly DecodeError on macs with a wrong protobuf. If you want smth else, add it to the exception tuple.